src/pages/tezos/claim-transaction.js

```diff
@@ -152,8 +153,17 @@ function Transaction({ location }) {
   const [user, setUser] = useAtom(userAtom);
   const [isUser] = useAtom(isUserAtom);
 
+  const [botTokenId, setTokenId] = useState(false);
+  const [copyLink, setCopyLink] = useState(false);
   const [claimButtonDisabled, setClaimButtonDisabledStatus] = useState(true);
 
+  useEffect(() => {
```

Comment:
Also is there's any better way to redirect, bot doesn't exist?

Reply:
I think redirecting users here will confuse them. It's better to show an error & a back button to go to the previous page.